### DISCUSSION TITLE
Copy data from Preview window to clipboard

### DISCUSSION THREAD

**Discussion by edhans** [TabularEditor/TabularEditor3#1093] [Ideas]
Allow a quick way to copy the entire dataset from the preview window to the clipboard. I can copy a record, but not the table.

**Comment by otykier** [comment]
Great suggestion, thanks! As a workaround for now, consider using the DAX query editor where a simple Ctrl+A followed by Ctrl+C will add the entire selection to the clipboard.

**Comment by edhans** [comment]
Perfect. Didn't realize the CTRL-A/C trick worked in the DAX Query view. I'll use that!
